src/FSharpLint.Client/LSPFSharpLintServiceTypes.fs

```diff
@@ -0,0 +1,70 @@
+﻿module FSharpLint.Client.LSPFSharpLintServiceTypes
+
+open System
+open System.Diagnostics
+open System.IO
+open StreamJsonRpc
+
+type FSharpLintResponseCode =
+    | ErrToolNotFound = -5
+    | ErrFileNotFound = -4
+    | ErrFilePathIsNotAbsolute = -3
+    | ErrCancellationWasRequested = -2
+    | ErrDaemonCreationFailed = -1
+    | OkCurrentDaemonVersion = 0
+
+type FSharpLintVersion = FSharpLintVersion of string
+type FSharpLintExecutableFile = FSharpLintExecutableFile of string
+type Folder = private Folder of string 
+with 
+    static member from (filePath: string) =
+        if File.Exists(filePath) then 
```

Comment:
@MrLuje should we call IsPathAbsolute here before this check? if it's not, throw an error

Reply:
oh wait, I just saw you are calling later Path.GetFullPath, so I'm guessing you're allowing relative paths here, however... (I continue on another line)

Reply:
It's true that an `IsPathAbsolute` call is already made before (https://github.com/MrLuje/FSharpLint/blob/843ef6deccbd1b5f399f572090536dc01c119d96/src/FSharpLint.Client/LSPFSharpLintService.fs#L151) but we could still have paths like `/foo/bar/baz/../../buzz` which is why there is a `Path.GetFullPath` here.
I have some like theses in added tests :)